docs/services/frame-system/_index.md

```diff
@@ -115,11 +115,126 @@ The resulting tree of reference frames looks like:
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### TransformPose
+
+Transform a given source Pose from the reference frame to a new specified destination which is a reference frame.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `query` [(`viam.proto.common.PoseInFrame`)](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame): The pose that should be transformed.
+- `destination` (str): The name of the reference frame to transform the given pose to.
+- `additional_transforms` (Optional[List[[viam.proto.common.Transform](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Transform)]]): A list of [additional transforms](#supplemental-transforms).
+
+**Returns:**
+
+- `PoseInFrame` [(PoseInFrame)](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame): Transformed pose in destination reference frame.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.transform_pose).
+
+```python {class="line-numbers linkable-line-numbers"}
+# Transform a pose into the frame of myArm
+first_pose = Pose(x=0.0, y=0.0, z=0.0, o_x=0.0, o_y=0.0, o_z=1.0, theta=0.0)
+first_pif = PoseInFrame(reference_frame="world", pose=first_pose)
+transformed_pif = await robot.transform_pose(first_pif, "myArm")
+print("Position: (x:", transformed_pif.pose.x, ", y:", transformed_pif.pose.y, ", z:", transformed_pif.pose.z, ")")
+print("Orientation: (o_x:", transformed_pif.pose.o_x,
+      ", o_y:", transformed_pif.pose.o_y,
+      ", o_z:",transformed_pif.pose.o_z,
+      ", theta:", transformed_pif.pose.theta, ")")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(`Context`)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `pose` (Optional[[referenceframe.PoseInFrame](https://pkg.go.dev/go.viam.com/rdk@v0.2.36/referenceframe#PoseInFrame)]): The pose that should be transformed.
+- `dst` (string): The name of the reference frame to transform the given pose to.
+- `additionalTransforms` (Optional[[referenceframe.LinkInFrame](https://pkg.go.dev/go.viam.com/rdk@v0.2.36/referenceframe#LinkInFrame)]): A list of [additional transforms](#supplemental-transforms).
+
+**Returns:**
+
+- `error` [(`error`)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+- `PoseInFrame` [(referenceframe.PoseInFrame)](https://pkg.go.dev/go.viam.com/rdk@v0.2.36/referenceframe#PoseInFrame): Transformed pose in destination reference frame.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/arm#Arm).
+
+```go {class="line-numbers linkable-line-numbers"}
+// Transform a pose into the frame of myArm
+firstPose := spatialmath.NewPoseFromPoint(r3.Vector{X: 0.0, Y: 0.0, Z: 0.0})
+firstPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, firstPose)
```

Comment:
```suggestion

// Transform firstPost into the world reference frame, the origin frame of myArm
firstPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, firstPose)
```

Reply:
It would be best to say something like the following, `// Establish the world as the reference for firstPose`

The second part would be better down by L228, where you could say something like `// Calculate firstPoseInFrame from the perspective of the origin frame of myArm` or thereabouts